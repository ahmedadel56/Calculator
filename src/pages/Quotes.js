import React from 'react';

const Quotes = () => (
  <div className="quotesBody">
    <h2>
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding. - William Paul Thurston
    </h2>
  </div>
);

export default Quotes;
