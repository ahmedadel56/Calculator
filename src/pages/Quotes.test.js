import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Quotes from './Quotes';

describe('<Quotes />', () => {
  it('it should match snapshot', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

