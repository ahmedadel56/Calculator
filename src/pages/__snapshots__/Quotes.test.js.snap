// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Quotes /> it should match snapshot 1`] = `
<div
  className="quotesBody"
>
  <h2>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </h2>
</div>
`;
